keyboards/bandominedoni/bandominedoni/keymaps/led/rules.mk

```diff
@@ -0,0 +1,2 @@
+RGB_MATRIX_ENABLE = yes       # Use RGB matrix (Don't enable this when RGBLIGHT_ENABLE is used.)
+RGB_MATRIX_DRIVER = WS2812    #
```

Comment:
This should go in the keyboard's rules.mk
```suggestion
```

Reply:
Thanks. Moved as you suggested.